### ISSUE TITLE
DXGKDDI_RENDER page is not rendering properly

### ISSUE THREAD

**Issue by shaboinkin** [MicrosoftDocs/windows-driver-docs-ddi#1025]
See screenshot. Happens on Safari and Firefox. 

<img width="1762" alt="Screen Shot 2021-01-13 at 10 30 27 AM" src="https://user-images.githubusercontent.com/1855146/104494461-5502e400-558b-11eb-81ce-eebfee4c2ed7.png">

This makes it hard to follow the information on the page.
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 35d49678-517e-cbe3-0020-5a58cb2e864d
* Version Independent ID: 1c91d8e4-f061-8cd5-eabb-3009ee4fd39c
* Content: [DXGKDDI_RENDER (d3dkmddi.h) - Windows drivers](https://docs.microsoft.com/en-us/windows-hardware/drivers/ddi/d3dkmddi/nc-d3dkmddi-dxgkddi_render)
* Content Source: [wdk-ddi-src/content/d3dkmddi/nc-d3dkmddi-dxgkddi_render.md](https://github.com/MicrosoftDocs/windows-driver-docs-ddi/blob/staging/wdk-ddi-src/content/d3dkmddi/nc-d3dkmddi-dxgkddi_render.md)
* Product: **windows-hardware**
* Technology: **windows-devices**
* GitHub Login: @lorihollasch
* Microsoft Alias: **loriwhip**

**Comment by EliotSeattle** [comment]
Thanks for the feedback, @shaboinkin 

Repro on Chrome and Edge, too, @lorihollasch .

**Comment by lorihollasch** [comment]
Thanks for reporting this, @shaboinkin  - I've fixed the formatting issue and all is good now.
